### DISCUSSION TITLE
Lesson 6, source .env error

### DISCUSSION THREAD

**Discussion by 0xDevGod** [Cyfrin/foundry-full-course-cu#681] [Q&A] [answered]
Source .env command giving this error

PRIVATE_KEY: command not found
SEPOLIA_RPC_URL: command not found

**Comment by shikhar229169** [comment, marked as answer]
Ensure that there are no spaces before and after the equal sign:
```
SEPOLIA_RPC_URL = your_url        <-  This is wrong
```
```
SEPOLIA_RPC_URL=your_url           <- Correct way
```

**Reply by 0xDevGod** [reply]
Thankyou so much, Shikhar! Btw why spaces causing error here?

**Reply by theirrationalone** [reply]
Bash catches exact matches only, so that was just an interpolation mismatch. `PRIVATE_KEY` ✅ not `PRIVATE_KEY ` extra white space after word `Y`. ❌

And you provided `$PRIVATE_KEY` into script but that was `$PRIVATE_KEY ` with an extra whitespace in the end. 

Same for (right hand side value)Rvalue after =
